src/lib/storage.js

```diff
@@ -55,7 +55,7 @@ class Storage extends ScratchStorage {
     }
     cacheDefaultProject () {
         const defaultProjectAssets = defaultProject(this.translator);
-        defaultProjectAssets.forEach(asset => this.cache(
+        defaultProjectAssets.forEach(asset => this.builtinHelper._store(
```

Comment:
We still need to do this to have the default project work offline. But I updated this call to avoid deprecation warnings.

Reply:
Is the plan to have the default project (and its assets) eventually live in storage like the `?` asset for example?

Reply:
We don't have a plan but we could think about that!